run_pipeline.py

```diff
@@ -90,11 +92,11 @@ def download_and_index(parser_args):
     logger.info("Testing to ensure Socrata index is stable or growing")
     test_index_growing(es, index_name)
 
-    output_file_name = 'complaints/ccdb/ccdb_output.json'
-    input_file_name = 'https://data.consumerfinance.gov/api/views/s6ew-h6mp/rows.json'
+    outfile = 'complaints/ccdb/ccdb_output.json'
```

Comment:
Will this need to respond to the infile / outfile option overwrite (like the above files) moving forward, or do we expect that because any non-data.consumerfinance.gov inputs will be more consistent that we will no longer need to run this check?

Reply:
Even better, we will not be using `run_pipeline` in the future.  It is only here until the Jenkins jobs can be switched over to `make`